demos/dotnet/dotnet/ProjectProperties.cs

```diff
@@ -17,15 +18,9 @@ class ProjectProperties
         public readonly List<string> CscOptions = new List<string>();
```

Comment:
Looks like this class is specialized to CSC... why not put the CSC extension methods in here, along with beefing up the name?